tests/dbgp/dbgpclient.php

```diff
@@ -111,13 +160,69 @@ function runTest( $data, array $commands, array $ini_options = null )
 			$i++;
 		}
 		fclose( $conn );
+		fclose( $ppipes[0] );
+		fclose( $ppipes[1] );
 		proc_close( $php );
 	}
 }
 
-function dbgpRun( $data, $commands, array $ini_options = null)
+class DebugClientIPv6 extends DebugClient
+{
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_AAAA);
+
+		if(!empty($records))
+		{
+			return $records[0]['ipv6'];
+		}
+
+		return "::1";
+	}
+
+	protected function getIP()
+	{
+		return "tcp://[" . $this->getIPAddress() . "]:" . $this->getPort();
+	}
+
+	public static function isSupported()
+	{
+		$ret = true;
+
+		if(!defined("AF_INET6"))
+			return false;
```

Comment:
Fixed the lines I found. Should the function brackets be on the same line or under the function statement as now?